mlir/utils/jenkins/Jenkinsfile.navi4x

```diff
@@ -0,0 +1,163 @@
+void buildProject(String target, String cmakeOpts) {
+    timeout(time: 60, activity: true, unit: 'MINUTES') {
+        cmakeBuild generator: 'Ninja',\
+            buildDir: 'build',\
+            buildType: 'RelWithDebInfo',\
+            installation: 'InSearchPath',\
+            steps: [[args: target]],\
+            cmakeArgs: """-DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
+              -DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang
+              ${cmakeOpts}"""
+    }
+}
+
+void showEnv() {
+    echo "$env.NODE_NAME"
+    sh 'cat /etc/os-release'
+    sh 'ulimit -a'
+    sh '/opt/rocm/bin/rocm-smi || true'
+    sh '/opt/rocm/bin/rocm_agent_enumerator'
+    sh 'id'
+    sh 'printenv'
+}
+
+String dockerArgs() {
+    return "--device=/dev/kfd --device=/dev/dri --group-add video --group-add render -v /etc/passwd:/etc/passwd:ro -v /etc/group:/etc/group:ro"
+}
+
+String dockerImage() {
+    // If this is being changed please change Dockerfile.migraphx-ci's base image as well
+    return 'rocm/mlir:rocm6.2-latest'
+}
+
+void setHeartbeat() {
+    script {
```

Comment:
Why all these `script {}` blocks? I'm pretty sure we don't need them?

Reply:
Not sure. Copied from our nightly. I assumed it was needed or at least wouldn't hurt. I will remove it.